tools/wptrunner/tox.ini

```diff
@@ -2,7 +2,7 @@
 xfail_strict=true
 
 [tox]
-envlist = py27-{base,chrome,edge,firefox,ie,opera,safari,sauce,servo}
+envlist = py27-{base,chrome,edge,firefox,ie,opera,safari,sauce,servo},py36-base
```

Comment:
Let's add `py37-base` here as well for those who don't don't have Python 3.6 installed (such as on Azure!).

Reply:
wptrunner doesn't work on 3.7 due to #16178